### PR TITLE
54 second step

### LINKED ISSUE #54: Second step
**Implement second step in registration complete.**

- In the second step , avatar download will be available. 

- Also add cropper for cut image to avatar type

### PR DESCRIPTION
**Pull request by Vova-Romaniuk**

### CODE REVIEW

File: StudCity.Frontend/src/UI/Avatar/index.jsx [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from "react";
+
+function Avatar({ src, className = "" }) {
+	return(
+		<div className={`rounded-full h-[180px] mx-auto w-[180px] overflow-hidden ${className}`}>
```
Comment (maryanchuk04) [inline StudCity.Frontend/src/UI/Avatar/index.jsx]: Maybe we can use h-44(176px) or h-48 (192px)

File: StudCity.Frontend/src/UI/fields/UploadImageInput.jsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+	};
+
+	return (
+		<div className="w-full flex">
+			<input 
+				type="file"
+				accept="image/*" 
+				ref={inputRef}
+				onChange={handleOnChange}
+				style={{ display: "none" }}
```
Comment (maryanchuk04) [inline StudCity.Frontend/src/UI/fields/UploadImageInput.jsx]: u can user "hidden" == display: none in tailwindcss

File: StudCity.Frontend/src/components/ImageCropper/index.jsx [hunk 3]
```diff
@@ -0,0 +1,52 @@
+import React, { useState } from "react";
+import Cropper from "react-easy-crop";
+import Button from "../../UI/Button";
```
Comment (maryanchuk04) [inline StudCity.Frontend/src/components/ImageCropper/index.jsx:3]: Add empty line

File: StudCity.Frontend/src/components/ImageCropper/index.jsx [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+		<div className="flex relative w-full h-full justify-end items-end ">
+				<Cropper
+					image={image}
+					aspect={aspectRatio}
+					crop={crop}
+					zoom={zoom}
+					onCropChange={setCrop} 
+					onZoomChange={setZoom}
+					onCropComplete={onCropComplete}
+					style={{
```
Comment (maryanchuk04) [inline StudCity.Frontend/src/components/ImageCropper/index.jsx:25]: Why you use inline styles? We have tailwind.

File: StudCity.Frontend/src/components/ImageCropper/index.jsx [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+						containerStyle: {
+							width: "400px",
+							height: "400px",
+							backgroundColor: "#fff",
+							margin: "auto",
+							bottom: "100px"
+						},
+					}}
+				/>
+				<div className="mx-auto h-fit gap-[20px]  mb-10 flex ">
```
Comment (maryanchuk04) [inline StudCity.Frontend/src/components/ImageCropper/index.jsx]: gap-5 == gap-[20px]

**Review by maryanchuk04 — CHANGES_REQUESTED** [review]

**Review by maria2908 — APPROVED** [review]
Nice job Kolega!

**Review by maryanchuk04 — APPROVED** [review]

**Pull request merged**